dev/com.ibm.ws.jsf.shared/resources/com/ibm/ws/jsf/shared/resources/messages.nlsprops

```diff
@@ -16,6 +16,8 @@
 #NLS_MESSAGEFORMAT_VAR
 # -------------------------------------------------------------------------------------------------
 
+# JSFG0100-JSFG0199 reserved for jsfContainer-2.X features
```

Comment:
Could this message be a complete sentence? Maybe add a verb after the subject and a period at the end?
JSFG0100-JSFG0199 is reserved for jsfContainer-2.X features.

Reply:
this is just a comment in the file, not an actual message